### ISSUE TITLE
LBP Dashboard: Swap Limit Error

### ISSUE THREAD

**Issue by BartuCulha** [PrimeDAO/prime-launch-dapp#638]

**Comment by dkent600** [comment]
@BartuCulha what does this mean: "Initiate swap for more than once. "

I just swapped .1 WETH and it worked fine.  Must be more specific repro steps.

**Comment by BartuCulha** [comment]
After a successful swap, please redo same process, and try to initiate a new swap

**Comment by dkent600** [comment]
Waiting for feedback from the Balancer TG channel

**Comment by dkent600** [comment]
There is a new little UI thingie:  If your expected rate of exchange changes, then I'm putting up a warning:

"Heads up that we just updated the exchange rate for D2D that you may receive"

**Comment by BartuCulha** [comment]
I am able to see the warning message and the button is disabled, however button is not grayed out. 

Please make the button grayed out until a new fetch comes along. 
@dkent600

**Comment by dkent600** [comment]
yes, this is what I mentioned I'm working on this morning and described in the comment just above.  However, the particular case you describe is now moot, and I am no longer going to prevent the swap from happening in this case.

**Comment by BartuCulha** [comment]
- [ ] Warning message to say: 

"Heads up that the exchange rate we're showing you for D2D has just changed by more than 1%"

**Comment by dkent600** [comment]
fixed
